proto/buf/alpha/audit/v1alpha1/event.proto

```diff
@@ -80,11 +100,135 @@ message Event {
   // Payload of the audited event. It specifies additional context on the event.
   oneof payload {
     PayloadOrganizationCreated organization_created = 6;
+    PayloadOrganizationDeleted organization_deleted = 7;
+    PayloadOrganizationMemberAdded organization_member_added = 8;
+    PayloadOrganizationMemberRoleChanged organization_member_role_changed = 9;
+    PayloadOrganizationMemberRemoved organization_member_removed = 10;
+    PayloadRepositoryCreated repository_created = 11;
+    PayloadRepositoryDeleted repository_deleted = 12;
+    PayloadRepositoryPushed repository_pushed = 13;
+    PayloadRepositoryMemberAdded repository_member_added = 14;
+    PayloadRepositoryMemberRoleChanged repository_member_role_changed = 15;
+    PayloadRepositoryMemberRemoved repository_member_removed = 16;
+    PayloadPluginCreated plugin_created = 17;
+    PayloadPluginDeleted plugin_deleted = 18;
+    PayloadPluginPushed plugin_pushed = 19;
+    PayloadUserRegistered user_registered = 20;
+    PayloadUserDeactivated user_deactivated = 21;
+    PayloadUserDeleted user_deleted = 22;
+    PayloadUserLoggedIn user_logged_in = 23;
+    PayloadUserLoggedOut user_logged_out = 24;
   }
 }
 
-// PayloadOrganizationCreated is the payload for an ORGANIZATION_CREATED event.
```

Comment:
chose to go comment-less on the names for these payloads as I think they're somewhat self-documenting, but happy to add in something